src/App.tsx

```diff
@@ -15,36 +22,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPrepearedGoods(
+  goods: string[],
+  options: { sortField: SortType | ''; isReversed: boolean },
+): string[] {
+  const prepearedSortGoods = [...goods];
+
+  if (options.sortField) {
+    prepearedSortGoods.sort((good1, good2) => {
+      switch (options.sortField) {
+        case SortType.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortType.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
```

Comment:
The `reset` functionality is not properly handled. When `SortType.Reset` is set, the `getPrepearedGoods` function should return the original order of goods, but currently, it does not handle `SortType.Reset`. Consider adding a case for `SortType.Reset` to return the goods in their original order.